src/Symfony/Component/Mime/Test/Constraint/EmailSubjectContains.php

```diff
@@ -0,0 +1,49 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Component\Mime\Test\Constraint;
+
+use PHPUnit\Framework\Constraint\Constraint;
+use Symfony\Component\Mime\Email;
+
+final class EmailSubjectContains extends Constraint
+{
+    private $expectedSubjectValue;
+
+    public function __construct(string $expectedSubjectValue)
+    {
+        $this->expectedSubjectValue = $expectedSubjectValue;
+    }
```

Comment:
Please use constructor property promotion, thanks

Reply:
Done pushed it